site/content/docs/5.1/examples/footers/index.html

```diff
@@ -9,186 +9,413 @@
-        <li class="nav-item"><a href="#" class="nav-link px-2 text-muted">About</a></li>
+  <footer class="mt-4 bg-dark">
+    <!-- Service links -->
+    <span class="visually-hidden">Service links</span>
+    <div class="w-100 px-2 px-md-4 py-3 navbar-dark">
+      <ul class="navbar-nav d-md-flex flex-md-row flex-wrap gap-3 gap-md-4">
+        <li><a class="nav-link d-flex align-items-center p-0 gap-1" href="#">
+          <svg width="1.875rem" height="1.875rem" focusable="false"><use xlink:href="#contact"/></svg>
+          Contact us
+          </a></li>
+        <li><a class="nav-link d-flex align-items-center p-0 gap-1" href="#">
+          <svg width="1.875rem" height="1.875rem" focusable="false"><use xlink:href="#locate"/></svg>
+          Locate a store
+          </a></li>
+        <li><a class="nav-link d-flex align-items-center p-0 gap-1" href="#">
+          <svg width="1.875rem" height="1.875rem" focusable="false"><use xlink:href="#coverage"/></svg>
+          Coverage map
+          </a></li>
+        <li><a class="nav-link d-flex align-items-center p-0 gap-1" href="#">
+          <svg width="1.875rem" height="1.875rem" focusable="false"><use xlink:href="#business"/></svg>
+          Business
+          </a></li>
+        <li><a class="nav-link d-flex align-items-center p-0 gap-1" href="#">
+          <svg width="1.875rem" height="1.875rem" focusable="false"><use xlink:href="#protection"/></svg>
+          Child protection
+          </a></li>
       </ul>
-      <p class="text-center text-muted">&copy; {{< year >}} Company, Inc</p>
-    </footer>
-  </div>
+    </div>
+  </footer>
 
   <div class="b-example-divider"></div>
 
-  <div class="container">
-    <footer class="row row-cols-1 row-cols-sm-2 row-cols-md-5 py-5 my-5 border-top">
-      <div class="col mb-3">
-        <a href="/" class="d-flex align-items-center mb-3 link-dark text-decoration-none">
-          <svg class="bi me-2" width="40" height="32"><use xlink:href="#bootstrap"/></svg>
-        </a>
-        <p class="text-muted">&copy; {{< year >}}</p>
-      </div>
-
-      <div class="col mb-3">
-
-      </div>
-
-      <div class="col mb-3">
-        <h5>Section</h5>
-        <ul class="nav flex-column">
-          <li class="nav-item mb-2"><a href="#" class="nav-link p-0 text-muted">Home</a></li>
-          <li class="nav-item mb-2"><a href="#" class="nav-link p-0 text-muted">Features</a></li>
-          <li class="nav-item mb-2"><a href="#" class="nav-link p-0 text-muted">Pricing</a></li>
-          <li class="nav-item mb-2"><a href="#" class="nav-link p-0 text-muted">FAQs</a></li>
-          <li class="nav-item mb-2"><a href="#" class="nav-link p-0 text-muted">About</a></li>
-        </ul>
-      </div>
+  <footer class="mt-4 bg-dark">
+    <!-- Terms & Conditions -->
+    <span class="visually-hidden">Terms and conditions</span>
+    <div class="w-100 px-2 px-md-4 py-3 navbar-dark">
+      <ul class="navbar-nav d-flex flex-md-row flex-wrap gap-4">
```

Comment:
I would say it is `gap-md-3 gap-4` because it is `20px` in desktop mode. Check the other lists to see if you can't handle a double mode (desktop / mobile)